core/controller/src/main/scala/whisk/core/controller/Controller.scala

```diff
@@ -135,6 +150,18 @@ object Controller {
 
     def optionalProperties = EntitlementProvider.optionalProperties
 
+    private def info(config: WhiskConfig, runtimes: Runtimes, apis: List[String]) = JsObject(
+        "description" -> "OpenWhisk".toJson,
+        "support" -> JsObject(
```

Comment:
Could also add a link to the StackOverflow items tagged with OpenWhisk (https://stackoverflow.com/questions/tagged/openwhisk).